src/test/java/com/nerodesk/om/aws/AwsDocsTest.java

```diff
@@ -29,17 +29,29 @@
  */
 package com.nerodesk.om.aws;
 
+import com.google.common.collect.Lists;
+import com.jcabi.s3.Bucket;
+import com.jcabi.s3.mock.MkBucket;
+import com.nerodesk.om.Docs;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
 import nl.jqno.equalsverifier.EqualsVerifier;
 import nl.jqno.equalsverifier.Warning;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 
 /**
  * Tests for {@link AwsDocs}.
  *
  * @author Krzysztof Krason (Krzysztof.Krason@gmail.com)
```

Comment:
@longtimeago I had to merge with master and krzyk had created the class with the equals test :)